tools/sof-kernel-log-check.sh

```diff
@@ -276,6 +276,13 @@ case "$platform" in
         # https://unix.stackexchange.com/questions/109294/mei-00000016-0-init-hw-failure
         ignore_str="$ignore_str"'|mei_me 0000:00:16\..: wait hw ready failed'
         ;;
+    adl)
+	# i915 AUX logs can be ignored
+        # origin logs seen on ADLS platforms
+        # i915 0000:00:02.0: [drm] *ERROR* AUX A/DDI A/PHY A: did not complete or timeout within 10ms (status 0xad4003ff)
+        # i915 0000:00:02.0: [drm] *ERROR* AUX A/DDI A/PHY A: not done (status 0xad4003ff)
+        ignore_str="$ignore_str"'|i915 [[:digit:].:]+: \[drm\] \*ERROR\* AUX .+'
```

Comment:
Is this tested? Regular expression is always tricky to match. Especially I'm not sure about `[[:digit:].:]+` will match for the numbers. If tested, I'm OK.